### ISSUE TITLE
Use a symbol for aliases that Yaml understands to be part of a string.

### ISSUE THREAD

**Issue by rec** [ManiacalLabs/BiblioPixel#836]
Right now we have to quote all our strings with aliases in them, because `@` is some special character for Yaml.

If there is no such character, then document this issue.

**Comment by adammhaile** [comment]
Not really... See here
https://symfony.com/doc/current/components/yaml/yaml_format.html
We could also support `alias.strip_test` or something where the work alias
itself is special and will attempt resolution if possible

**Comment by rec** [comment]
> Strings containing any of the following characters must be quoted. 

There are characters not in that list!

I just wrote a test to see how it actually works - https://github.com/rec/test/blob/master/test_yml.py  It's consistent with your document, and seems to show that we can choose between the following characters to introduce aliases:

    $()+-./:;<=?\^_~

We can eliminate some of these right off the bat, because the alias occurs in an Address, where the characters `()/.=+-_\\` can potentially occur, which leaves us with

    $:;<?^~

I'm going to eliminate `:;<` as "too confusing" so that leaves us with

    $?^~

We used to have `$` as the alias character, but it was annoying when using command line arguments, because `$` expands inside `"` quotes.  However, it's quite likely no one is using those arguments.

`^` has some attractiveness because you don't even have to quote it if you use it in a command line. 

Honestly, I think `$` is the only solution.  Everyone understands it means "expand this symbol".
